pytest_xvfb.py

```diff
@@ -132,6 +133,12 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     if config.getoption('--no-xvfb') or not xvfb_available():
         config.xvfb = None
+        if (sys.platform.startswith('linux')
+                and 'DISPLAY' in os.environ
+                    and not config.getoption('--no-xvfb')):
+                        print(
+                            'pytest-xvfb could not find Xvfb. '
+                            'You can install it to avoid windows from opening.')
```

Comment:
This is not like the example in pep8 I quoted :wink:

I meant:

```python
        if (sys.platform.startswith('linux')
                and 'DISPLAY' in os.environ
                and not config.getoption('--no-xvfb')):
            print('pytest-xvfb could not find Xvfb. '
                  'You can install it to avoid windows from opening.')
```

That way, the body of the `if` is indented as usual (four spaces), and the contiunation lines of the `if` are simply indented one more, to tell them apart from the body easily.

Also, I'm no native english speaker, but is "to avoid windows from opening" correct english? Wouldn't "to prevent windows from being shown" or so make more sense?